.ci/jobs/elastic+helm-charts+master+integration-elasticsearch.yml

```
@@ -25,8 +25,11 @@
         #!/usr/local/bin/runbld
         set -euo pipefail
 
+        source /usr/local/bin/bash_standard_lib.sh
+
         set +x
-        export VAULT_TOKEN=$(vault write -field=token auth/approle/login role_id="$VAULT_ROLE_ID" secret_id="$VAULT_SECRET_ID")
+        VAULT_TOKEN=$(retry 5 retry 5 vault write -field=token auth/approle/login role_id="$VAULT_ROLE_ID" secret_id="$VAULT_SECRET_ID")
```

Oops, extra vim `p` in here? `retry 5 retry 5`

```suggestion
        VAULT_TOKEN=$(retry 5 vault write -field=token auth/approle/login role_id="$VAULT_ROLE_ID" secret_id="$VAULT_SECRET_ID")
```

I _knew_ I wasn't just a useless rubber stamp on these PRs :baseball:

---

Looks like my vim macro wasn't idempotent. I'm curious whether this would have still worked and just retried 25 times...

---

Thank you for catching! ⚾️